google-cloud-error_reporting/lib/google/cloud/error_reporting/rails.rb

```diff
@@ -117,8 +117,10 @@ def self.merge_rails_config rails_config
             config.credentials ||= config.keyfile
             config.credentials ||= er_config.credentials || er_config.keyfile
             config.credentials ||= gcp_config.credentials || gcp_config.keyfile
-            config.service_name ||= er_config.service_name
-            config.service_version ||= er_config.service_version
+            config.service_name ||= \
+              (er_config.service_name || gcp_config.service_name)
+            config.service_version ||= \
+              (er_config.service_version || gcp_config.service_version)
```

Comment:
The top-level config `service_name` and `service_version` are used here, correct? But the final commit appears to remove them.

Reply:
The config is not removed. The commit moves the configuration from the google-cloud-core gem to the google-cloud-debugger gem.

Reply:
But this usage here references the top-level config from error_reporting, correct?

Reply:
nm, I found the comments on the outdated code, looks good.